### PR TITLE
[Storybook] Introduce TypingIndicator into storybook v8

### PR DESCRIPTION
**Pull request by JoshuaLai**
# What
image
image

# Why
Migrate typing indicator to storybook v8

# How Tested
navigate to package/storybook8 
run rushx start

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/storybook8/stories/Components/TypingIndicator/Docs.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { Canvas, Meta, Source } from '@storybook/blocks';
+import TypingIndicatorExampleText from '!!raw-loader!./snippets/TypingIndicator.snippet.tsx';
+import CustomizeStyleExampleText from '!!raw-loader!./snippets/CustomizeStyle.snippet.tsx';
+import CustomizeUserRenderingExampleText from '!!raw-loader!./snippets/CustomizeUserRendering.snippet.tsx';
+import * as TypingIndicatorStories from './index.stories';
+
+<Meta of={TypingIndicatorStories} /># Typing Indicator Component to notify the local user when one or more participants
```
Comment (AmyL219) [inline packages/storybook8/stories/Components/TypingIndicator/Docs.mdx:7]: Should `# Typing Indicator Component to notify the local user when one or more participants
Comment` be wrapped in another new line

Comment (JoshuaLai) [reply packages/storybook8/stories/Components/TypingIndicator/Docs.mdx]: Odd looks like a linter forced this edit. Fixed the formatting

**Review by AmyL219 — APPROVED** [review]

File: packages/storybook8/stories/Components/TypingIndicator/index.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+export const CustomizeUserRenderingExampleDocsOnly = {
+  render: CustomUserRenderExample
+};
+
+const meta: Meta = {
+  title: 'Components/Typing Indicator',
+  component: TypingIndicatorComponent,
+  argTypes: {
+    // Hidden Controls
```
Comment (prabhjot-msft) [inline packages/storybook8/stories/Components/TypingIndicator/index.stories.tsx:28]: add typing users from "controls to add" to fix the naming from `typingUsers` to `Typing Users`

Comment (JoshuaLai) [reply packages/storybook8/stories/Components/TypingIndicator/index.stories.tsx:29]: Added

**Review by prabhjot-msft — APPROVED** [review]

**Pull request merged**